packages/amplify-codegen-appsync-model-plugin/src/__tests__/visitors/__snapshots__/appsync-java-visitor.test.ts.snap

```diff
@@ -1374,9 +1395,12 @@ public final class privateType implements Model {
 "
 `;
 
-exports[`AppSyncModelVisitor Model with Auth should generate models with public authorization 1`] = `
+exports[`AppSyncModelVisitor Model with Auth should generate class with public authorization 1`] = `
 "package com.amplifyframework.datastore.generated.model;
 
+import com.amplifyframework.core.model.AuthStrategy;
```

Comment:
Can you make sure imports are sorted alphabetically?  These should be in the section below with the other `com.amplifyframework.core.model.` imports.

Reply:
Moved the imports to be with the other com.amplifyframework.core.model imports